lib/services/episode.ts

```diff
@@ -0,0 +1,68 @@
+import type { Tables } from '@/types/supabase/database';
+
+import { z } from 'zod';
+
+import type { PodcastIndexEpisodeType } from './podcast-index/fetch-episodes';
+
+import { DatabaseError } from '../errors';
+import { createSupabaseServiceClient } from './supabase/service';
+
+const serializedEpisodesSchema = z.array(
+  z.object({
+    audio_url: z.string().min(1),
+    description: z.string().nullable(),
+    duration: z.number().positive(),
+    episode_number: z.number().nullable(),
+    image: z.string(),
+    podcast_index_guid: z.string().min(1),
+    published_date: z.coerce.date(),
+    published_date_pretty: z.string().min(1),
```

Comment:
**question:** do we need this? I guess we'll prettify dates in the UI with `date-fns` anyways 💅

Reply:
actually, you're right. I just want to keep it for now to have better understanding about imported episodes in the DB view. Basically, if they're coming in with descending date order or not. After some time trying the import mechanism, I will get rid of this column.

Reply:
![datetime](https://github.com/experiment-station/beecast/assets/27828319/3004cfa6-2830-4e63-981e-55b2c68a9c7d)